### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR includes new cal csvs for instruments returning from vendor, as well as deployment-specific cal csvs for METBK Loggers on surface moorings and ADCPs on profiler moorings.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 16 AM Updates [DT-127]](https://trello.com/c/8zHulE3Q/127-pioneer-16-am-updates-dt-127)

**Comment by rtravis-whoi** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/ADCPSL/CGINS-ADCPSL-21393__20210329.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21393,CC_scale_factor1,0.45,constant; CP04OSPM-00014
+21393,CC_scale_factor2,0.45,constant
+21393,CC_scale_factor3,0.45,constant
+21393,CC_scale_factor4,0.45,constant
+21393,CC_bin_size,800.0,in cm
+21393,CC_dist_first_bin,1695.0,in cm
+21393,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (asmith-whoi) [inline calibration/ADCPSL/CGINS-ADCPSL-21393__20210329.csv:8]: OK

File: calibration/ADCPTG/CGINS-ADCPTG-19151__20210329.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+19151,CC_scale_factor1,0.45,constant; CP02PMCI-00015
+19151,CC_scale_factor2,0.45,constant
+19151,CC_scale_factor3,0.45,constant
+19151,CC_scale_factor4,0.45,constant
+19151,CC_bin_size,400.0,in cm
+19151,CC_dist_first_bin,901.0,in cm
+19151,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (asmith-whoi) [inline calibration/ADCPTG/CGINS-ADCPTG-19151__20210329.csv:8]: OK

File: calibration/ADCPTG/CGINS-ADCPTG-20495__20210329.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20495,CC_scale_factor1,0.45,constant; CP02PMCO-00015
+20495,CC_scale_factor2,0.45,constant
+20495,CC_scale_factor3,0.45,constant
+20495,CC_scale_factor4,0.45,constant
+20495,CC_bin_size,400.0,in cm
+20495,CC_dist_first_bin,901.0,in cm
+20495,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (asmith-whoi) [inline calibration/ADCPTG/CGINS-ADCPTG-20495__20210329.csv:8]: OK

File: calibration/METBKA/CGINS-METLGR-00008__20210329.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR008,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP03ISSM-00013
+LGR008,CC_height_of_air_humidity_measurement_m,4.255,
+LGR008,CC_height_of_air_temperature_measurement_m,4.255,
+LGR008,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR008,CC_jcool,1,Constant
+LGR008,CC_jwarm,1,Constant
+LGR008,CC_zinvpbl,600,Constant
+LGR008,CC_use_velpt,1,
```
Comment (asmith-whoi) [inline calibration/METBKA/CGINS-METLGR-00008__20210329.csv:9]: OK

File: calibration/METBKA/CGINS-METLGR-00010__20210329.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR010,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP01CNSM-00014
+LGR010,CC_height_of_air_humidity_measurement_m,4.255,
+LGR010,CC_height_of_air_temperature_measurement_m,4.255,
+LGR010,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR010,CC_jcool,1,Constant
+LGR010,CC_jwarm,1,Constant
+LGR010,CC_zinvpbl,600,Constant
+LGR010,CC_use_velpt,1,
```
Comment (asmith-whoi) [inline calibration/METBKA/CGINS-METLGR-00010__20210329.csv:9]: OK

File: calibration/METBKA/CGINS-METLGR-00017__20210329.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR017,CC_depth_of_conductivity_and_temperature_measurements_m,1.11,CP01CNSM-00014
+LGR017,CC_height_of_air_humidity_measurement_m,4.255,
+LGR017,CC_height_of_air_temperature_measurement_m,4.255,
+LGR017,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR017,CC_jcool,1,Constant
+LGR017,CC_jwarm,1,Constant
+LGR017,CC_zinvpbl,600,Constant
+LGR017,CC_use_velpt,1,
```
Comment (asmith-whoi) [inline calibration/METBKA/CGINS-METLGR-00017__20210329.csv:9]: OK

File: calibration/METBKA/CGINS-METLGR-00037__20210329.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR037,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP04OSSM-00013
+LGR037,CC_height_of_air_humidity_measurement_m,4.255,
+LGR037,CC_height_of_air_temperature_measurement_m,4.255,
+LGR037,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR037,CC_jcool,1,Constant
+LGR037,CC_jwarm,1,Constant
+LGR037,CC_zinvpbl,600,Constant
+LGR037,CC_use_velpt,1,
```
Comment (asmith-whoi) [inline calibration/METBKA/CGINS-METLGR-00037__20210329.csv:9]: OK

**Review by asmith-whoi — APPROVED** [review]

File: calibration/FLORTD/CGINS-FLORTD-01299__20210212.csv [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+1299,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1299_Calibration_Files_2021-02-12.zip > BBFL2W-1299 Charsheets.pdf
+1299,CC_dark_counts_cdom,43,
+1299,CC_dark_counts_chlorophyll_a,48,
+1299,CC_dark_counts_volume_scatter,48,
+1299,CC_depolarization_ratio,0.039,Constant
+1299,CC_measurement_wavelength,700,Constant
+1299,CC_scale_factor_cdom,0.0521,
+1299,CC_scale_factor_chlorophyll_a,0.0123,
+1299,CC_scale_factor_volume_scatter,2.201E-06,
+1299,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/FLORTD/CGINS-FLORTD-01299__20210212.csv:11]: Looks good

File: calibration/METBKA/CGINS-METLGR-00008__20210329.csv (hunk 4 above)
Comment (cdobs) [reply calibration/METBKA/CGINS-METLGR-00008__20210329.csv:9]: Looks good, verified SN in config sheet

File: calibration/METBKA/CGINS-METLGR-00010__20210329.csv (hunk 5 above)
Comment (cdobs) [reply calibration/METBKA/CGINS-METLGR-00010__20210329.csv:9]: Looks good, verified SN in config sheet

File: calibration/METBKA/CGINS-METLGR-00037__20210329.csv (hunk 7 above)
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00037__20210329.csv:9]: Looks good, verified SN in config sheet

File: calibration/METBKA/CGINS-METLGR-00017__20210329.csv (hunk 6 above)
Comment (cdobs) [reply calibration/METBKA/CGINS-METLGR-00017__20210329.csv:9]: Looks good, verified SN in config sheet

File: calibration/PHSEND/CGINS-PHSEND-P0157__20210302.csv [hunk 9]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0157,CC_ea434,17372.0,source_file PHSEN-D_SAMI-pH_SN_P0157_Validation_Certificate_2021-03-02.pdf
+P0157,CC_ea578,94.1,
+P0157,CC_eb434,2284.0,
+P0157,CC_eb578,38676.0,
+P0157,CC_ind_off,0,
+P0157,CC_ind_slp,1,
+P0157,CC_psal,35,Constant
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PHSEND/CGINS-PHSEND-P0157__20210302.csv:8]: Looks good

File: calibration/PHSENF/CGINS-PHSENF-P0141__20210302.csv [hunk 10]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0141,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0141_Validation_Certificate_2021-03-02.pdf
+P0141,CC_ea578,94.1,
+P0141,CC_eb434,2284.0,
+P0141,CC_eb578,38676.0,
+P0141,CC_ind_off,0,
+P0141,CC_ind_slp,1,
+P0141,CC_psal,35,Constant
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PHSENF/CGINS-PHSENF-P0141__20210302.csv:8]: Looks good

File: calibration/PRESFB/CGINS-PRESFB-01392__20210112.csv [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+26P-1392,CC_pressure_offset_calibration_coefficient,-2.094000e-01,
+26P-1392,CC_slope_correction_factor,1,Constant
+26P-1392,CC_t1,2.761045e+01,
+26P-1392,CC_t2,2.362710e-01,
+26P-1392,CC_t3,1.669386e+01,
+26P-1392,CC_t4,3.230367e+01,
+26P-1392,CC_u0,5.843641e+00,
+26P-1392,CC_y1,-3.952915e+03,
+26P-1392,CC_y2,-1.037722e+04,
+26P-1392,CC_y3,0.000000e+00,
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PRESFB/CGINS-PRESFB-01392__20210112.csv:19]: looks good

File: calibration/PRESFB/CGINS-PRESFB-01386__20200315.csv [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+26P76848-1386,CC_pressure_offset_calibration_coefficient,-2.832000e-01,
+26P76848-1386,CC_slope_correction_factor,1,constant
+26P76848-1386,CC_t1,2.763804e+01,
+26P76848-1386,CC_t2,4.278750e-01,
+26P76848-1386,CC_t3,1.776240e+01,
+26P76848-1386,CC_t4,2.186445e+01,
+26P76848-1386,CC_u0,5.810283e+00,
+26P76848-1386,CC_y1,-3.944121e+03,
+26P76848-1386,CC_y2,-1.073763e+04,
+26P76848-1386,CC_y3,0.000000e+00,
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PRESFB/CGINS-PRESFB-01386__20200315.csv:19]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01062__20210222.csv [hunk 13]
```diff
@@ -0,0 +1,8 @@
...
...272.74, 273.55, 274.37, 275.18, 275.99, 276.81, 277.62, 278.43, 279.25, 280.06, 280.87, 281.69, 282.5, 283.31, 284.13, 284.94, 285.75, 286.57, 287.38, 288.2, 289.01, 289.82, 290.64, 291.45, 292.26, 293.08, 293.89, 294.7, 295.52, 296.33, 297.14, 297.96, 298.77, 299.58, 300.4, 301.21, 302.02, 302.84, 303.65, 304.46, 305.28, 306.09, 306.9, 307.71, 308.53, 309.34, 310.15, 310.97, 311.78, 312.59, 313.4, 314.22, 315.03, 315.84, 316.65, 317.46, 318.28, 319.09, 319.9, 320.71, 321.52, 322.34, 323.15, 323.96, 324.77, 325.58, 326.39, 327.2, 328.02, 328.83, 329.64, 330.45, 331.26, 332.07, 332.88, 333.69, 334.5, 335.31, 336.12, 336.93, 337.74, 338.55, 339.36, 340.17, 340.98, 341.79, 342.6, 343.41, 344.21, 345.02, 345.83, 346.64, 347.45, 348.26, 349.06, 349.87, 350.68, 351.49, 352.3, 353.1, 353.91, 354.72, 355.52, 356.33, 357.14, 357.94, 358.75, 359.55, 360.36, 361.17, 361.97, 362.78, 363.58, 364.39, 365.19, 366, 366.8, 367.6, 368.41, 369.21, 370.02, 370.82, 371.62, 372.42, 373.23, 374.03, 374.83, 375.63, 376.44, 377.24, 378.04, 378.84, 379.64, 380.44, 381.24, 382.04, 382.84, 383.64, 384.44, 385.24, 386.04, 386.84, 387.64, 388.44, 389.24, 390.04, 390.83, 391.63, 392.43, 393.23, 394.02, 394.82]",
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/NUTNRB/CGINS-NUTNRB-01062__20210222.csv:8]: Spot check - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01063__20210222.csv [hunk 14]
```diff
@@ -0,0 +1,8 @@
...
...28, 275.09, 275.9, 276.72, 277.53, 278.34, 279.15, 279.96, 280.77, 281.58, 282.39, 283.2, 284.01, 284.82, 285.63, 286.44, 287.26, 288.07, 288.88, 289.69, 290.5, 291.31, 292.12, 292.93, 293.74, 294.55, 295.36, 296.17, 296.98, 297.79, 298.6, 299.41, 300.22, 301.03, 301.84, 302.65, 303.46, 304.27, 305.08, 305.89, 306.7, 307.51, 308.31, 309.12, 309.93, 310.74, 311.55, 312.36, 313.17, 313.98, 314.78, 315.59, 316.4, 317.21, 318.02, 318.83, 319.63, 320.44, 321.25, 322.06, 322.86, 323.67, 324.48, 325.29, 326.09, 326.9, 327.71, 328.51, 329.32, 330.12, 330.93, 331.74, 332.54, 333.35, 334.15, 334.96, 335.77, 336.57, 337.38, 338.18, 338.99, 339.79, 340.59, 341.4, 342.2, 343.01, 343.81, 344.61, 345.42, 346.22, 347.02, 347.83, 348.63, 349.43, 350.23, 351.04, 351.84, 352.64, 353.44, 354.24, 355.05, 355.85, 356.65, 357.45, 358.25, 359.05, 359.85, 360.65, 361.45, 362.25, 363.05, 363.85, 364.65, 365.45, 366.24, 367.04, 367.84, 368.64, 369.44, 370.23, 371.03, 371.83, 372.62, 373.42, 374.22, 375.01, 375.81, 376.6, 377.4, 378.2, 378.99, 379.78, 380.58, 381.37, 382.17, 382.96, 383.75, 384.55, 385.34, 386.13, 386.93, 387.72, 388.51, 389.3, 390.09, 390.88, 391.67, 392.46, 393.25, 394.04, 394.83, 395.62]",
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/NUTNRB/CGINS-NUTNRB-01063__20210222.csv:8]: Spot check - looks good

**Review by cdobs — APPROVED** [review]
Everything looks good

**Pull request merged**